pkg/render/apiserver_test.go

```diff
@@ -218,6 +218,13 @@ var _ = Describe("API server rendering tests", func() {
 		Expect(d.Spec.Template.Spec.Volumes[1].ConfigMap.Items[0].Key).To(Equal("config"))
 		Expect(d.Spec.Template.Spec.Volumes[1].ConfigMap.Items[0].Path).To(Equal("policy.conf"))
 		Expect(len(d.Spec.Template.Spec.Volumes[1].ConfigMap.Items)).To(Equal(1))
+
+		clusterRole := GetResource(resources, "tigera-network-admin", "", "rbac.authorization.k8s.io", "v1", "ClusterRole").(*rbacv1.ClusterRole)
+		Expect(clusterRole.Rules).To(ConsistOf(networkAdminPolicyRules))
+
+		clusterRole = GetResource(resources, "tigera-ui-user", "", "rbac.authorization.k8s.io", "v1", "ClusterRole").(*rbacv1.ClusterRole)
+		Expect(clusterRole.Rules).To(ConsistOf(uiUserPolicyRules))
```

Comment:
Not sure if this description is still accurate (about what resources to expected in the render): 
https://github.com/tigera/operator/pull/777/files#diff-969ab7a6ef1d9769dd6b6d7977f3300fR87-R100

If it's accurate could you update it with the new ClusterRoles you added? If it's not accurate, I think we should just remove it.